ci-operator/config/openshift-priv/cluster-network-operator/openshift-priv-cluster-network-operator-master.yaml

```diff
@@ -206,6 +206,40 @@ tests:
       NETWORK_TYPE: OVNKubernetes
       TEST_SUITE: openshift/conformance/parallel
     workflow: openshift-e2e-openstack-ipi
+- as: e2e-aws-live-migration-sdn-ovn
+  optional: true
+  steps:
+    cluster_profile: aws
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
+      OVERRIDE_OPENSHIFT_SDN_DEPRECATION: "true"
+      TEST_SDN_LIVE_MIGRATION_OPTIONS: target-cni=OVNKubernetes
+      TEST_SUITE: openshift/network/live-migration
+    test:
+    - ref: ovn-sdn-migration-feature-migration-config-egress-firewall
+    - ref: ovn-sdn-migration-feature-migration-config-multicast
+    - ref: openshift-e2e-test
+    - ref: ovn-sdn-migration-feature-migration-validate-egress-firewall
+    - ref: ovn-sdn-migration-feature-migration-validate-multicast
+    workflow: openshift-e2e-aws-sdn
+  timeout: 5h0m0s
```

Comment:
passing rehearsal job [here](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/46805/rehearse-46805-pull-ci-openshift-ovn-kubernetes-release-4.16-e2e-aws-live-migration-sdn-ovn/1747557555996987392) only took a little over 2h. can we reduce this?